### PR TITLE
Added possibility to log structured messages

### PR DESCRIPTION
**Pull request by dowgird**

### CODE REVIEW

File: logger/logevent.go [hunk 1]
```diff
@@ -55,10 +55,15 @@ func (v Severity) String() string {
...
-	Message   string            `json:"message"`
-	Labels    map[string]string `json:"-"`
-	CallDepth int               `json:"-"`
-	Severity  Severity          `json:"-"`
+	Message string `json:"message"`
+	// If present, this will be set as Payload to Cloud Logging
+	// instead of Message and LocalTimeStamp.
+	//
+	// Note: Message is still sent to local logs.
+	StructuredMessage interface{}       `json:"omitempty"`
```
Comment (adjackura) [inline logger/logevent.go:63]: StructuredPayload, Message is a magic field that the UI uses to populate the default view.

**Comment by adjackura** [comment]
/hold
/assign hopkiw

Liam can you take a look, this change is so that we can log arbitrary structured payloads but still fully maintain backwards compatibility.

**Comment by dowgird** [comment]
Friendly ping.

File: logger/logevent.go (hunk 1 above)
Comment (dowgird) [reply logger/logevent.go]: Renamed.

**Review by adjackura — APPROVED** [review]

**Pull request merged**